test-case/verify-kernel-boot-log.sh

```diff
@@ -18,10 +18,16 @@ set -e
 # shellcheck source=case-lib/lib.sh
 source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
 
-start_test
-# TODO: replace start_test with this:
-# trap 'print_test_result_exit $?' EXIT
-# https://github.com/thesofproject/sof-test/issues/1112
+trap 'boot_log_exit_handler $?' EXIT
+
+boot_log_exit_handler()
+{
+    local exit_code=$1
+
+    journalctl --boot > "$LOG_ROOT"/boot_log.txt
```

Comment:
Normally I don't pay attention to user space. What can we expect more clues from user space log?

Reply:
> What can we expect more clues from user space log?

- Sound daemons errors (pulseaudio, pipewire,...)
- Display / auto-login issues: https://github.com/thesofproject/sof-test/discussions/964, https://github.com/thesofproject/sof-test/discussions/998
- Anything else that could give clues.

If you look at some examples, you'll see the added volume is low compared to SOF debug logs. One example on a fresh boot:

```
root@jf-lnlm-rvp-nocodec-3:~# journalctl -k | wc -l
1857
root@jf-lnlm-rvp-nocodec-3:~# journalctl -b | wc -l
2313
```

Reply:
A couple more after looking at `diff -u <(journalctl -k) <(journalctl -b) | less`

- Network initialization
- NTP init
- sensors stuff (thermal)
- other systemd logs